src/intl/messages.js

```diff
@@ -699,8 +726,10 @@ export const messages: { [messageId: string]: MessageType } = {
   utilizationNoDataAvailableMessage: 'Utilization data is only available when the VM is running.',
   utilizationNoHistoricData: 'No historic data available',
   utilizationNoNetStats: 'Network utilization is not currently available for this VM.',
+  usbAutoshare: 'USB Auto-Share',
   useBrowserBelow: 'Please use one of the browsers below.',
   useCtrlAltEnd: 'Use Ctrl+Alt+End',
+  useLocalDrives: 'Use Local Drives',
```

Comment:
This is not required/planned as part of console options

Reply:
Added just in case to be compliant with Web Admin console options.

Reply:
Same here

Reply:
technically we support it (but it's disabled):
https://github.com/oVirt/ovirt-web-ui/blob/53f33a8dc918b23e0f63498c4049fc70f9c794ab/src/sagas/console/rdpBuilder.js#L155